src/governance/TimelockController.vy

```diff
@@ -5,26 +5,32 @@
 @license GNU Affero General Public License v3.0 only
 @author pcaversaccio
 @custom:coauthor cairoeth
-@notice This module allows for timelocking operations by scheduling and
-        executing transactions. By leveraging AccessControl, the 
-        TimelockController introduces three roles: proposer, executor,
-        and canceller. The proposer role is responsible for proposing
-        operations, the executor is responsible for executing scheduled
-        proposals, and the canceller is responsible for cancelling
-        proposals. The owner is the sole admin of the roles and can grant
-        and revoke them. In the constructor, proposers will be granted
+@notice This module enables the timelocking of operations by scheduling
+        and executing transactions. By leveraging `AccessControl`, the
+        `TimelockController` contract introduces three roles:
+            1. proposer (`PROPOSER_ROLE`),
+            2. executor (`EXECUTOR_ROLE`), and
+            3. canceller (`CANCELLER_ROLE`).
+        The proposer role is responsible for proposing operations, the executor
+        role is responsible for executing scheduled proposals, and the canceller
+        is responsible for cancelling proposals. This contract is self-administered
+        by default (unless an optional admin account is granted at construction),
+        meaning administration tasks (e.g. grant or revoke roles) have to go through
+        the timelock process. At contract creation time, proposers are granted
         the proposer and canceller roles.
 
-        Proposals must be scheduled with a delay that is greater than or
-        equal to the minimum delay (`minDelay()`), which can be updated via
-        a proposal to itself and is measured in seconds. Additionally,
-        proposals can be associated with preceding proposals, which must be
-        executed before the proposal can be executed. Ready proposals can be
-        executed by the executor, who is solely responsible for calling the
-        `execute()` function.
+        The proposals must be scheduled with a delay that is greater than or equal
+        to the minimum delay 'get_minimum_delay', which can be updated via a proposal
+        to itself and is measured in seconds. Additionally, proposals can be linked
+        to preceding proposals that must be executed before the proposal can be executed.
         
-        Proposals can be timelocked individually or in batches. The latter is
-        useful for operations that require execution in the same block.
+        Ready proposals can be executed by the executor, who is solely responsible
+        for calling the `execute` function. Eventually, the proposals can be batched
```

Comment:
executor can also call `executeBatch`, so we may want to add this function next to the `execute` function reference

Reply:
Resolved via https://github.com/pcaversaccio/snekmate/pull/195/commits/b852bc5a4da232430974a9bc1356fca27bd68a75.